python/cuml/dask/neighbors/kneighbors_regressor.py

```diff
@@ -61,18 +57,14 @@ class KNeighborsRegressor(NearestNeighbors):
     verbose : int or boolean, default=False
         Sets logging level. It must be one of `cuml.common.logger.level_*`.
         See :ref:`verbosity-levels` for more info.
-    output_type : {'input', 'cudf', 'cupy', 'numpy', 'numba'}, default=None
-        Variable to control output type of the results and attributes of
-        the estimator. If None, it'll inherit the output type set at the
-        module level, `cuml.global_output_type`.
-        See :ref:`output-data-type-configuration` for more info.
     """
     def __init__(self, client=None, streams_per_handle=0,
-                 verbose=False, **kwargs):
+                 verbose=False, batch_size=1024, **kwargs):
```

Comment:
Since we are now specifying these arguments in multiple places, we should define a constant to centralize the default for all places that use it so we only need to update it in one place. Maybe inside dask.neighbors.NearestNeighbors since everything depends on that? We should also increase the default to a larger number. 

This value isn't the most straightforward to set and is pretty use-case specific, but it should definitely be raised to a value much larger than 1024. I would propose something like `2000000` perhaps, which with an input of 500 features would only use about 4GB `(2000000*500*4)`.